.github/workflows/create_cache_command.yml

```diff
@@ -97,12 +97,10 @@ jobs:
       platforms: linux/${{ matrix.platform }}
       dockerfile: build/devdeps.Dockerfile
       toolchain: ${{ matrix.toolchain }}
-      build_args: |
-        llvm_commit=${{ needs.metadata.outputs.llvm_commit }}
-        pybind11_commit=${{ needs.metadata.outputs.pybind11_commit }}
```

Comment:
No, the submodule commit is not updated, and more work may be needed to update pybind to the latest (for one, there was an issue with the latest pybind if I remember correctly from last time I tried, and we also have customized the docs generation a bit, so that would need to be updated). I would like to defer any package updates to a separate PR.